vminfo_parser/vminfo_parser.py

```diff
@@ -717,6 +801,13 @@ def main(*args: t.Optional[str]) -> None:  # noqa: C901
     if config.prod_env_labels:
         environments = config.prod_env_labels.split(",")
 
+    if config.sort_by_site:
+        site_dataframe = vm_data.add_site_specific_columns()
+        analyzer.cli_output.print_site_usage("Memory", site_dataframe)
+        analyzer.cli_output.print_site_usage("CPU", site_dataframe)
+        analyzer.cli_output.print_site_usage("Disk", site_dataframe)
+        analyzer.cli_output.print_site_usage("VM", site_dataframe)
```

Comment:
Could this be better achieved by passing a list to print_site_usage and having that function iterate over it, rather than calling it 4 times in a row.

Reply:
good catch. Changed this to a list